### DISCUSSION TITLE
[Idea] Add possibility to change position of feature values

### DISCUSSION THREAD

**Discussion by micka-fdz** [PrestaShop/PrestaShop#32267] [Ideas]
#### Is your feature request related to a problem?

It's currently impossible to change position of features values.
This feature could be useful to change display order on product page or with `ps_facetedsearch` module.

#### Describe the solution you'd like

We need to add possibility to change position of feature values, like attribute values.

**Comment by hibatallahAouadni** [comment]
Hello @micka-fdz 


Thanks for your issue. Unfortunately, there is not enough information provided to reproduce it and work on it. **You must follow the template to submit a complete bug report**. 

Read more about how we expect the issues to be handled [here](https://build.prestashop.com/news/how-bug-reports-are-handled/). Feel free to create a new issue if you can provide further information, we will be glad to investigate it.

Thanks!

**Comment by micka-fdz** [comment]
Hi @hibatallahAouadni, I updated my issue description.

It's not about a bug report but a feature request.

**Comment by hibatallahAouadni** [comment]
Hello @micka-fdz,

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 

Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know! 

Thank you

**Comment by yannicka** [comment]
Just to give an idea of the need for this feature, I note here that this feature would be very handy for a client I have.

**Comment by d33x** [comment]
+1! Should be a must-have.

It is not logical not to be able to sort in some cases (like size):
![03-12-2021 11-15-23](https://user-images.githubusercontent.com/76649472/144586142-d098ee94-d195-4a93-8d40-5ac1969c7beb.png)

**Comment by amigainc** [comment]
I agree, for imported data from other databases or CSV files, it should be great to have the same position than _Features_ have. See below:
![position](https://user-images.githubusercontent.com/6299136/175085010-c1b20650-58f8-4983-a525-195256a195c8.png)

**Comment by micka-fdz** [comment]
I will try to make a PR soon.

**Comment by palmequipment** [comment]
It is odd that attribute values are repositionable but feature values are not in Prestashop 1.7.2
Controlling the order of feature values would help greatly with presenting product features in a logical order.

**Comment by SmartPhoneLover** [comment]
Is there some update about this implementation?
